doc/onboarding.md

```diff
@@ -100,18 +108,22 @@ onboarding session.
         (especially if it just has nits left).
   * Approving a change
     * Collaborators indicate that they have reviewed and approve of the
-        the changes in a pull request by commenting with `LGTM`, which stands
-        for "looks good to me".
-    * You have the power to `LGTM` another collaborator's (including TSC/CTC
-        members) work.
-    * You may not `LGTM` your own pull requests.
-    * You have the power to `LGTM` anyone else's pull requests.
-
-  * What belongs in node:
-    * opinions vary, but I find the following helpful:
-    * if node itself needs it (due to historic reasons), then it belongs in node
-      * that is to say, url is there because of http, freelist is there because of http, et al
-    * also, things that cannot be done outside of core, or only with significant pain (example: async-wrap)
+        the changes in a pull request using Github’s approval interface
+    * Some people like to comment `LGTM` (“Looks Good To Me”)
```

Comment:
> It's mostly for anyone who solely uses the "approved" list for the "reviewed by" list. It's better than nothing.

Hmm okay, but doesn't that mean that those people will never record your approval at all?